.github/workflows/test.yml

```diff
@@ -85,17 +107,25 @@ jobs:
           # django versions that support py3.10
           - python-version: '3.10'
             django-version: 3.2
```

Comment:
3.2 is still supported, i believe.
move it to `functional`?

Reply:
It has nothing to do with functional tests; it's for Django. Django 3.2 is the first version to support Python 3.10.

Reply:
but this is under `functional-legacy-py:`